tools/release/backport_pr.sh

```diff
@@ -24,59 +24,100 @@ ensure_command () {
   fi
 }
 
+display_usage () {
```

Comment:
Nit: Have you considered using a single expression for this? Either echoing a single multiline string to stderr or `cat > /dev/stderr -<< EOF` should work.

Reply:
Much better stylistically. Switched.